app/javascript/practice-select.js

```diff
@@ -0,0 +1,23 @@
+import Vue from 'vue'
+import PracticeSelect from './practice-select.vue'
+
+document.addEventListener('DOMContentLoaded', () => {
+  const selector = '#js-practice-select'
+  const practiceSelect = document.querySelector(selector)
+  const matchedTitle = location.search.match(/&title=.+$/g)
+  const title = matchedTitle
+    ? decodeURI(matchedTitle).replace('&title=', '')
+    : ''
```

Comment:
@konaga-k 
正しいタイトル名に`+`がないということを前提にしたら、以下のような感じですかね:eyes:

```suggestion
  const title = matchedTitle
    ? decodeURI(matchedTitle).replace('&title=', '').replace(/\+/g, ' ')
    : ''
```

Reply:
@gentamura 
57d4279a27564206353d0b91b5515182bafcfb8c
ありがとうございます！
ご確認お願いします🙏

Reply:
よさそうです〜🙆‍♀️